### PR TITLE
Add username to user

### PR DESCRIPTION
**Pull request by skrix**

### CODE REVIEW

File: app/decorators/order_decorator.rb [hunk 1]
```diff
@@ -6,9 +6,9 @@ class OrderDecorator < ApplicationDecorator
   delegate :order_meals, to:     :object,
                          prefix: true
 
-  delegate :email, to:        :user,
-                   allow_nil: true,
-                   prefix:    true
+  delegate :username, :email, to:        :user,
```
Comment (lessuk400) [inline app/decorators/order_decorator.rb]: one line will look prettier

File: app/models/user.rb [hunk 2]
```diff
@@ -7,7 +7,8 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  validates :email, uniqueness: true
+  validates :email, :username, uniqueness: true
+  validates :email, :username, presence:   true
```
Comment (lessuk400) [inline app/models/user.rb]: validates :email, :username, uniqueness: true, presence: true

File: spec/features/lunch_admin_create_menu_spec.rb [hunk 3]
```diff
@@ -6,13 +6,20 @@
...
     sign_in lunch_admin
 
     visit new_menu_path
 
     compose_menu
   end
 
+  def set_day_to_monday
+    monday = Date.today.beginning_of_week
+    Timecop.travel(monday)
```
Comment (andrewr224) [inline spec/features/lunch_admin_create_menu_spec.rb]: Are you sure these two shouldn't be in reverse order?

File: spec/features/lunch_admin_create_menu_spec.rb [hunk 4]
```diff
@@ -6,13 +6,20 @@
...
+
     sign_in lunch_admin
 
     visit new_menu_path
 
     compose_menu
   end
 
+  def set_day_to_monday
+    monday = Date.today.beginning_of_week
```
Comment (andrewr224) [inline spec/features/lunch_admin_create_menu_spec.rb]: It works today, but will it do it's job tomorrow? 
Select a day that is definitely a Monday

**Comment by andrewr224** [comment]
Ok, now you should improve your user edit future spec to check for a new user name

**Pull request merged**